src/EditorFeatures/Core/Shared/Utilities/BrushToColorConverter.cs

```diff
@@ -1,39 +0,0 @@
-// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Globalization;
-using System.Windows.Data;
-using System.Windows.Media;
-
-namespace Microsoft.CodeAnalysis.Editor.Shared.Utilities
-{
-    internal class BrushToColorConverter : IValueConverter
```

Comment:
This file was copied/pasted but is available from Microsoft.VisualStudio.PlatformUI which we now get as a NuGet package.